tests/fast-integration/utils/index.js

```diff
@@ -602,6 +603,18 @@ function waitForJob(name, namespace = "default", timeout = 900000, success = 1)
   );
 }
 
+async function kubectlExecInPod(pod, container, cmd, namespace = "default") {
```

Comment:
I couldn't get `exec` from the Kubernetes javascript client to work so instead doing this fallback to kubectl.